components/dashboard/src/data/git-providers/unified-repositories-search-query.ts

```diff
@@ -128,15 +128,12 @@ export function deduplicateAndFilterRepositories(
     if (results.length === 0) {
         // If the searchString is a URL, and it's not present in the proposed results, "artificially" add it here.
         if (isValidGitUrl(searchString)) {
-            console.log("It's valid man");
```

Comment:
These super rad debug logs I accidentally left in with https://github.com/gitpod-io/gitpod/pull/20287. Yikes!